app/ui/src/app/import-export/import/import.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { NotificationType } from 'patternfly-ng';
+import { IntegrationStore, IntegrationSupportService } from '@syndesis/ui/store';
+import { Integrations } from '@syndesis/ui/model';
+import { NotificationService } from '@syndesis/ui/common';
+import {
+  FileUploader,
+  FileItem
+} from 'ng2-file-upload';
+
+@Component({
+  selector: 'syndesis-import-integration-component',
+  templateUrl: './import-integration.component.html',
+  styleUrls: ['./import-integration.component.scss']
+})
+export class ImportIntegrationComponent implements OnInit {
+  public uploader: FileUploader;
```

Comment:
In TypeScript all members are public by default, so this accessor is redundant. not wrong, but redundant.